README.md

```diff
@@ -1,30 +1,80 @@
-# Brzeczyk
-A compiler implemented by TCS UJ students at "Compilers" course in the 2022/23 winter semester.
+# Brzęczyk
 
-## Building
-Project can be built using `gradle`. Installation of `gradle` is not required. To build, you can run
-```bash
-./gradlew build
-```
+[ˈbʒɛ̃n͇ʧ̑ɨk]
 
-## Testing
-Tests can be run with
-```bash
-./test_all.sh
-```
+A simple programming language with very *polished* syntax.
+
+Implemented by students at Theoretical Computer Science Department of Jagiellonian University
```

Comment:
```suggestion
Implemented by the students at the Theoretical Computer Science Department of the Jagiellonian University
```

Reply:
"the" before "students" would sound as if we were the only.